src/promptflow/promptflow/executor/flow_validator.py

```diff
@@ -34,10 +34,11 @@ def _ensure_nodes_order(flow: Flow):
                     continue
                 if i.value not in dependencies:
                     msg = (
```

Comment:
Would you like to rename `msg` to `message_format`?

Reply:
done